doc/news/_2016-12-17_website_release.md

```diff
@@ -0,0 +1,118 @@
+changed to achieve this goal and an equal amount of effort was put into
+writing a system that decouples the documentation structure on the
+website from the structure used within the Elektra repository.
+
+The tutorials section was partially reworked to make the first steps
+together with Elektra easier for our users. Clearly the effort put into
+the tutorials is worth it. Thanks to <insert name of krit0n> for that!
+
+We should note though that the structure on the website is not really
+final yet and will definitely develop over time, especially the bindings
+and libraries sections will get some more attention.
+
+If you are interested in what techniques we use to structure our files,
+you can have a look at the
+[rest-frontend readme](https://blob.libelektra.org/src/tools/rest-frontend/README.md).
+
+### Homepage & News ###
+
+Besides the documentation we also wanted a place to properly present
+ourselves and our news around Elektra. For this reason we created a new
+home page which shall give an overview of what Elektra is and can do.
+Additionally to that we also added a news section.
+
+We hope that you enjoy our new appearance as much as we do!
+
+### Snippet Sharing ###
+
+Another important part of the website and also without doubt the part
+that took most effort to create, is the service that allows for sharing
+of configuration snippets. It is run by a REST service fully built with
+the help of [CppCMS](http://http://cppcms.com/) on basis of Elektra as
+data store. All data concerning snippets and user accounts is stored
+in Elektras key database (of course with password being properly hashed).
+
+The service allows to paste configuration snippets in various (supported)
+formats and to tag, describe and name them. This in return allows for easy
+search of snippets by keywords and to download them in other formats than
+the upload format as well.
+
+Clearly the service is meant to be driven by its users. Therefore we ask
+you to share your own configuration snippets, maybe they can be of help
+or simply a time saver for someone else!
+
+### NoScript ###
+
+The website is fully written with the help of AngularJS and is therefore
+heavily based on JavaScript. This should be no issue though as the
+website does only use resources that can be found in the official Elektra
+repository. So in case you cannot or do not want to use JavaScript, you
+can find all resources also [here](https://git.libelektra.org).
+
+## Feedback ##
+
+At this point there is not much more to say about the new website except for:
+Feel free to explore it!
+
+We greatly appreciate all feedback, be it for the website, the snippet sharing
+service or other parts of the Elektra project. We always have an open ear
+for suggestions and we also like to help with technical issues, simply
+[leave us a note on github](https://bugs.libelektra.org)!
+
+## Stay tuned! ##
+
+Subscribe to the
+[RSS feed](http://www.libelektra.org/rss/feed.rss)
```

Comment:
Yes, would be much better (rss -> news). The permalink below rss is a bit strange anyway, and obviously we do not want people already subscribed to have any issues. (The permalinks are often linked externally, so they really should work, too)

I thought it was unchanged because I looked into the other NEWS files, but it seemed like you changed it in all news files.